README.md

```diff
@@ -1,91 +1,90 @@
+  concerns than is possible with stock CloudFormation without resorting to
+  Lambda-backed custom resources. See the [pre-processor documentation
   below](#yaml-pre-processing) for more details.
 * It has bash command completion support.
 
 ## Pronunciation
 
-iidy is pronounced "eye-dee", like the audience's response to Cab Calloway in Minnie the
-Moocher: https://www.youtube.com/watch?v=8mq4UT4VnbE&feature=youtu.be&t=50s
+iidy is pronounced "eye-dee", like the audience's response to Cab Calloway in
+[Minnie the Moocher](https://www.youtube.com/watch?v=8mq4UT4VnbE&feature=youtu.be&t=50s).
 
 ## Demo
-[![asciicast](https://asciinema.org/a/8rzW1WyoDxMdVJpvpYf2mHm8E.png)](https://asciinema.org/a/8rzW1WyoDxMdVJpvpYf2mHm8E)
 
-Here's a comparison between developer experience of `iidy` and `Ansible`
-when you run into errors with a CloudFormation stack:
-[![asciicast](https://asciinema.org/a/jExZ5S8Pk4KzlJiufGoll0rXs.png)](https://asciinema.org/a/jExZ5S8Pk4KzlJiufGoll0rXs?t=45)
+[![asciicast](https://asciinema.org/a/8rzW1WyoDxMdVJpvpYf2mHm8E.png)](https://asciinema.org/a/8rzW1WyoDxMdVJpvpYf2mHm8E)
 
 ## Installation
 
-### MacOS: Binary Install via Homebrew
+iidy is distributed as a self-contained executable (via [pkg](https://github.com/zeit/pkg)).
+
+### Binary Install for macOS via Homebrew
 
-Use Unbounce's custom Homebrew Tap to install iidy.  This is the preferred method for macos.
+Use Unbounce's custom Homebrew Tap to install iidy. This is the preferred method for macOS.
 
-```Shell
+```shell
 brew tap unbounce/homebrew-taps
 brew update
 brew install iidy
 ```
 
 ### Binary Installation on Other Platforms
-```Shell
+
+```shell
 # Grab the appropriate binary from the releases page.
-wget https://github.com/unbounce/iidy/releases/download/v1.6.6-rc1/iidy-linux-amd64.zip
-# or wget https://github.com/unbounce/iidy/releases/download/v1.6.6-rc1/iidy-macos-amd64.zip
+wget https://github.com/unbounce/iidy/releases/download/v1.6.7/iidy-linux-amd64.zip
+# or wget https://github.com/unbounce/iidy/releases/download/v1.6.7/iidy-macos-amd64.zip
 unzip iidy*.zip
 chmod +x iidy
-mv iidy /usr/local/bin/   # or somewhere more appropriate
+mv iidy /usr/local/bin/ # or somewhere more appropriate
 ```
 
 ### Installation from Source
 
 Counter-intuitively, this requires more disk space than the binary
 install. You need Node 7+ and npm 4+ installed.
 
-```Shell
+```shell
 git clone git@github.com:unbounce/iidy.git
 cd iidy
-npm install . && npm run build # to compile our source first
+npm install && npm run build # to compile our source first
 ln -s $(pwd)/bin/iidy /usr/local/bin/
 # or npm install -g .
 ```
 
+### Docker Image
+
+A Docker image is automatically published to [Docker
+Hub](https://hub.docker.com/r/unbounce/iidy). The `latest` tag is updated on
+merges to the `master` branch. Releases are tagged with their version number.
+
+```shell
+docker run unbounce/iidy:v1.6.7
+```
+
 ## Usage
 
 ### Help
-```
+
+```shell
```

Comment:
We should skip the `shell` here as the syntax highlighting is incorrect in the output.